hack/mdl-style.rb

```diff
@@ -3,6 +3,7 @@
 #Refer below url for more information about the markdown rules.
 #https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
 
+rule 'MD007', :indent => 4
```

Comment:
Note: This is added here to make it easy to align markdown in a code editor (hence 4 spaces). As this PR itself may evolve and be merged much later, if folks agree I can split this change out to a different PR.

@BenamarMk @raghavendrabhat @vdeenadh @tjanssen3